airflow/providers/weaviate/hooks/weaviate.py

```diff
@@ -190,7 +190,12 @@ def get_collection(self, name: str) -> Collection:
         return client.collections.get(name)
 
     def delete_collections(
-        self, collection_names: list[str] | str, if_error: str = "stop"
+        self,
+        collection_names: list[str] | str,
+        by_property: list[str] | str | None = None,
+        contains_any: list[Any] | str | None = None,
+        contains_all: list[Any] | str | None = None,
+        if_error: str = "stop",
```

Comment:
```suggestion
        if_error: str = "stop",
        *,
        by_property: list[str] | str | None = None,
        contains_any: list[Any] | str | None = None,
        contains_all: list[Any] | str | None = None,
```

The order change might break some user code

Reply:
Oh sorry! Will do the change asap